jablib/src/main/java/org/jabref/logic/quality/consistency/BibliographyConsistencyCheck.java

```diff
@@ -8,17 +8,62 @@
 import java.util.Map;
 import java.util.SequencedCollection;
 import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 import java.util.function.BiConsumer;
 
+
 import org.jabref.logic.bibtex.comparator.BibEntryByCitationKeyComparator;
 import org.jabref.logic.bibtex.comparator.BibEntryByFieldsComparator;
 import org.jabref.logic.bibtex.comparator.FieldComparatorStack;
 import org.jabref.model.entry.BibEntry;
 import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.SpecialField;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.entry.field.UserSpecificCommentField;
 import org.jabref.model.entry.types.EntryType;
 
 public class BibliographyConsistencyCheck {
 
+    // ----------------------------------------------------------------
+    // Fields that we do *not* want to check for consistency:
+    //   – every field in StandardField.AUTOMATIC_FIELDS
+    //   – COMMENT, CROSSREF, CITES, PDF, REVIEW, SORTKEY, SORTNAME, TYPE, XREF, GROUPS
+    //   – plus any SpecialField or UserSpecificCommentField (filtered at runtime)
+    // ----------------------------------------------------------------
+    private static final Set<Field> FILTERED_FIELDS = Stream
+            // start with JabRef’s built‐in AUTOMATIC_FIELDS set
+            .concat(
+                    StandardField.AUTOMATIC_FIELDS.stream(),
+                    // then add the extra fields to ignore
+                    Stream.of(
+                            StandardField.COMMENT,
+                            StandardField.CROSSREF,
+                            StandardField.CITES,
+                            StandardField.PDF,
+                            StandardField.REVIEW,
+                            StandardField.SORTKEY,
+                            StandardField.SORTNAME,
+                            StandardField.TYPE,
+                            StandardField.XREF,
+                            StandardField.GROUPS
+                    )
+            )
+            .collect(Collectors.toSet());
+
+    /**
+     * Drop any field we don’t want to check.
+     */
+    private static Set<Field> filterFields(Collection<Field> fields) {
+        return fields.stream()
+                     .filter(f -> !FILTERED_FIELDS.contains(f))
+                     .filter(f -> !(f instanceof SpecialField))
+                     .filter(f -> !(f instanceof UserSpecificCommentField))
```

Comment:
We avoid abbreviations
```suggestion
                     .filter(field -> !FILTERED_FIELDS.contains(field))
                     .filter(field -> !(field instanceof SpecialField))
                     .filter(field -> !(field instanceof UserSpecificCommentField))
```